Gemfile

```diff
@@ -7,6 +7,7 @@ platforms :rbx do
   gem 'rubinius-developer_tools'
 end
 
-gem 'country_select', '~> 2.5.2'
-gem 'railties'
+gem 'activemodel', '~> 5.2.0.rc1'
```

Comment:
Why we need to have it in this Gemfile as well?

Reply:
Hey @gobijan, I think that you should remove that line

Reply:
Had this one in because otherwise I couldn't bundle update. You have a better solution?

09:43 $ bundle update
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies....
Bundler could not find compatible versions for gem "actionpack":
  In Gemfile:
    railties (~> 5.2.0.rc1) was resolved to 5.2.0.rc1, which depends on
      actionpack (= 5.2.0.rc1)

    simple_form was resolved to 3.5.1, which depends on
      actionpack (< 6, > 4)

Bundler could not find compatible versions for gem "activemodel":
  In Gemfile:
    simple_form was resolved to 3.5.1, which depends on
      activemodel (< 6, > 4)

Could not find gem 'activemodel (< 6, > 4)', which is required by gem 'simple_form', in any of the sources.

Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    simple_form was resolved to 3.5.1, which depends on
      activemodel (< 6, > 4) was resolved to 4.0.1, which depends on
        activesupport (= 4.0.1)

    railties (~> 5.2.0.rc1) was resolved to 5.2.0.rc1, which depends on
      activesupport (= 5.2.0.rc1)

Reply:
You can run the bundle just for the specific version, something like:

```
BUNDLE_GEMFILE=$PWD/gemfiles/Gemfile.rails-5-2-stable bundle install
```

Reply:
I'm not sure if we have to set the version of railties in our main Gemfile, the bundle update is failing because that.

@tegon @rafaelfranca some suggestions?

Reply:
@gobijan I think we don't need to have this change. If you remove the `railties` version and run bundle install it will download the last version available that from now it's 5.2.

We need just to keep the Gemfile.rails-5-2-stable .

Could you do this change? then we can merge it 😄